prql-compiler/tests/integration-vendors/docker-compose.yml

```
@@ -0,0 +1,39 @@
+version: "3"
+
+services:
+  postgres:
+    image: "postgres:15-alpine"
```

Is there any reason to specify version 15?

```suggestion
    image: "postgres:alpine"
```

---

No specific reason. I just took the latest version

---

Does dependabot upgrade these? If it does, that's ideal — then we lock versions but they don't fall out of date.

(BTW fine to refine these sorts of things in another PR)

---

I don't know if dependabot checks these files but the latest version should be fine